src/utils/get-site.ts

```diff
@@ -12,7 +12,13 @@ export const getSiteByName = async (api, siteName) => {
     }
 
     return siteFoundByName
-  } catch {
-    error('Site not found. Please rerun "netlify link"')
+  } catch (error_) {
+    // @ts-expect-error TS(2571) FIXME: Object is of type 'unknown'.
+    if (error_.status === 401) {
+      // @ts-expect-error TS(2571) FIXME: Object is of type 'unknown'.
```

Comment:
Fix these TS errors with pulling in this PR https://github.com/netlify/cli/pull/6877.

Reply:
Fixed!